docs/docs/recipes/working-with-themes.md

```diff
@@ -4,6 +4,72 @@ title: "Recipes: Working with Themes"
 
 A [Gatsby theme](/docs/themes/what-are-gatsby-themes) abstracts Gatsby configuration (shared functionality, data sourcing, design) into an installable package. This means that the configuration and functionality isn’t directly written into your project, but rather versioned, centrally managed, and installed as a dependency. You can seamlessly update a theme, compose themes together, and even swap out one compatible theme for another.
 
+## Creating a new site using a theme
+
+Found a theme you'd like to use on your project? Awesome! You can work through setting up by following the steps below.
+
+> If you'd like to take a look at more theme options, you could take a look at this [list of themes](https://www.npmjs.com/search?q=gatsby-theme) .
+
+### Prerequisities
+
+The [Gatsby CLI](/docs/gatsby-cli) installed
+
+### Directions
+
+1. Create a gatsby site
+
+```shell
+gatsby new {your-project-name}
+```
+
+2. Change directory and install theme
```

Comment:
```suggestion
2. Change directory and install theme

In this example, our theme is `gatsby-theme-blog`. You can replace that reference with whatever your theme is named.
```

Thoughts on doing it this way instead of having the additional example snippet?

Reply:
Gotcha! I agree.